examples/jsm/materials/MeshGouraudMaterial.js

```diff
@@ -270,8 +270,6 @@ const GouraudShader = {
 
 			#endif
 
-			#include <lightmap_fragment>
```

Comment:
Doesn't this removal break light map support in `MeshGouraudMaterial`?

Reply:
Yes, but I wonder why it supports it while others don't. I would either inline it here like in the basic material, or include the chunk in the basic. As the only thing depending on this chunk is an example.

Reply:
> Yes, but I wonder why it supports it while others don't.

`MeshGouraudMaterial` uses per vertex shading so a different approach than all lit materials in the core which use per fragment shading. Hence, the shader chunks differ.

> include the chunk in the basic. 

The code isn't compatible because of the `RECIPROCAL_PI` usage in the basic shader.

Custom shader code might depend on the chunk so a removal could cause unwanted noise on app level. TBH, I'm not it's worth to risk that just because of a minor clean up.

Reply:
Just to explain my reasoning - I’m trying to normalize the shader chunks from 77 to 162 in my app. I fetch them all transform them and get a JS file with two shaders and then try to process them to add my own hook syntax. 
A few other maps have foomap_thing but lightmap is the only one inlined in the source. So I guess moving forward from 162 it would be just ever so slightly more consistent. But then, are these libraries even going to stay for long? It doesn’t fix the inconsistency retroactively. 
I think the argument of it being a dependency is fair, but that is kinda the whole problem I’m dealing with :) 
I pulled all the tags and compared them (templates) to each other. Some times they are intact for ten versions other times it’s a few versions in a row. My ocd probably kicked in here, I’d move the PI into the chunk, have the goraud inline it if it’s needed.

Reply:
@pailhead Do you use `MeshGouraudMaterial`? It has not been properly updated and has deficiencies at this point.

Regardless, I think the chunk you are concerned about can be inlined.

Reply:
No, I just didn’t find any other mention of the chunk. I figured since it’s an example, that maybe it got stale. I’m just trying to figure out if I can get a set of rules to manipulate the template. This one first got me thinking if it’s even needed on the basic material. Since it’s flat shaded and I’m building a node editor, one could build their own aoMap, lightMap, modulate themselves and just plug altogether with albedo into the diffuse channel. So in my case I want to strip away this chunk altogether. 

Again it’s very very minor, I doubt anyone uses these paths l, but just caught my eye.

Reply:
> Regardless, I think the chunk you are concerned about can be inlined.

Fair enough! Since the chunk is not a common one, let's hope there won't be any friction on app level.

Reply:
So remove it but add it back in to the goraud example as code?

Reply:
Yes. The PR in its current state isn't complete.